client/starwhale/base/bundle_copy.py

```diff
@@ -55,6 +56,24 @@ class _UploadPhase:
     CANCEL = "CANCEL"
 
 
+@unique
+class FileType(Enum):
+    MANIFEST = "MANIFEST"
+    SRC_TAR = "SRC_TAR"
+    MODEL = "MODEL"
+    DATA = "DATA"
+
+
+@dataclass
+class FileDesc:
+    path: Path
+    name: str
+    size: int
+    file_type: FileType
+    signature: str = ""
+    existed: bool = False
```

Comment:
is `existed` property the same level for other FileDesc properties?

Reply:
`existed` is a dynamic property?

Reply:
yes, it is dynamic, existed files are returned by the first http request:"_do_ubd_bundle_prepare", then via override function 'upload_files' to get the files located at local FS, finally to filter the files

Reply:
updated with fileDesc.path.exists()